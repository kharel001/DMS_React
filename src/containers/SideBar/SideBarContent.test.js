import React from 'react';
import SideBarContent from './SideBarContent';
import {
    configure,
    shallow
} from 'enzyme';
import {IntlProvider} from 'react-intl';
import EnzymeAdapter from 'enzyme-adapter-react-16';
// JestHook.mock('expo-font');
import { MemoryRouter } from 'react-router-dom';

import { Provider } from 'react-redux';
import configureStore from 'redux-mock-store';
import {act} from 'react-dom/test-utils';
import { tsConstructorType } from '@babel/types';

const mockStore = configureStore([]);

configure({
    adapter: new EnzymeAdapter
})

// const itemClass = new Item();
describe('<SideBarContent />', () => {
it("Components SideBarContent", () => {
    let wrapper = shallow(
                < SideBarContent>
                </SideBarContent>
)
    expect(wrapper).toMatchSnapshot()
})
})
