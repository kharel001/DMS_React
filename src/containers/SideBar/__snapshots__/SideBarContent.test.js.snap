// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SideBarContent /> Components SideBarContent 1`] = `ShallowWrapper {}`;
